### DISCUSSION TITLE
Is it possible to change the underlying representation of memory?

### DISCUSSION THREAD

**Discussion by rkshthrmsh** [ferrilab/bitvec#259] [Q&A]
Let's say I have a `BitVec<u8, Msb0>` of `len()` `16`. Is there a way to convert the first 12 bits (`..12`) into a `u32` value?

**Comment by jngls** [comment]
You can use the `load()` and `store()` functions of the `Bitfield` trait: eg. `let val: u32 = bit_vec[0..12].load()` and `bit_vec[0..12].store(val)`.
